cmd/juju/application/config.go

```diff
@@ -35,8 +35,9 @@ const (
 displayed if no key is specified.
 
 By default, any configuration changes will be applied to the currently active
-generation. A specific generation can be targeted by using the --generation
-option with a value of "current" or "next".
+model branch. A specific branch can be targeted by using the --branch option. 
+When working on an active branch, changes can be applied to the model in the
+normal way by specifying "master" as the option value.
```

Comment:
I'm not sure about what the "normal way" is, can we be explicit here?

Reply:
Luckily, there was an intermittent test failure in the merge invoked before this.

Care to suggest prose?

Reply:
@mitechie might have opinions :wink:

Reply:
I don't think we want to call it "model branch". I'd just say "currently active branch". 

Can we have it so that this text only appears if the flag is set?


My suggestion would be:

  By default, any configuration changes will be applied to the currently active
branch A specific branch can be specified using the `--branch` option. Immediate changes can be made to the model by specifying ` --branch=master`.